tasks/css-tools.js

```diff
@@ -26,22 +56,15 @@ const wrapCSSResult = (content) => {
     `;
 };
 
-const __dirname = path.dirname(fileURLToPath(import.meta.url));
-const nodeModulesDir = path.resolve(__dirname, '..', 'node_modules');
-const configPath = path.resolve(path.join(__dirname, '..', 'config'));
-let header;
-try {
-    header = fs.readFileSync(path.join(configPath, 'license.js'), 'utf8');
-} catch (error) {
-    throw new Error(error);
+const licensePath = path.resolve(__dirname, '..', 'config', 'license.js');
+let header = '';
+if (fs.existsSync(licensePath)) {
+    header = fs.readFileSync(licensePath, 'utf8');
+    header = header.replace('<%= YEAR %>', new Date().getFullYear());
 }
 
-header = header.replace('<%= YEAR %>', new Date().getFullYear());
-
 export const processCSS = async (cssPath) => {
```

Comment:
Feel free to ignore if it is out of scope, but could we check in `build-css.js` if there is any benefit if we parallelize calling this?

pseudo-code:
`Promise.all(cssPaths.map((cssPath) => processCSS(cssPath)));`

Reply:
Okay, going to flag this one as out-of-scope since we're not really making functional changes to css-tools at this time, just a little clean-up.